frontend/src/components/metadata/MetadataDefinitionEntry.tsx

```diff
@@ -131,6 +157,17 @@ export function MetadataDefinitionEntry() {
 						flexDirection: "row",
 					}}
 				>
+					<Button
+						variant="contained"
+						aria-label="edit"
+						onClick={() => {
+							setEditMetadataDefinitionOpen(true);
+						}}
+						endIcon={<EditIcon />}
+						sx={{ float: "right" }}
+					>
+						Edit
+					</Button>
```

Comment:
<img width="284" alt="image" src="https://github.com/clowder-framework/clowder2/assets/13950475/74d75e89-c1aa-42f7-849a-1fd2787df08b">

Could you add a little bit margin between these two buttons?

Reply:
done.